torchgeometry/image/sobel.py

```diff
@@ -112,6 +112,17 @@ def spatial_gradient(input: torch.Tensor) -> torch.Tensor:
     return SpatialGradient()(input)
 
 
+def spatial_gradient_2nd_order(input: torch.Tensor) -> torch.Tensor:
```

Comment:
we have laplace operator. Can we use it for this ?

Reply:
Not in current form: we need to have 1d gradients for gxx and gyy, while current laplacian is 2d

Reply:
ok, gotcha. Do you think there's any optimization to be applied here to bootstrap 2nd order ?